src/audio/host.c

```diff
@@ -76,6 +76,7 @@ struct host_data {
 	uint32_t host_period_bytes;
 	uint16_t stream_tag;
 	uint16_t no_stream_position; /**< 1 means don't send stream position */
+	uint8_t cont_update_posn; /**< 1 means continuous update stream position */
```

Comment:
suggested commit message edit:

The position is currently updated by the host component when a period was consumed. The granularity of this position is too large, applications relying on timer-based scheduling may query the position at any time and conformance test suites fail.

This patch suggests a more precise position reporting, by storing the latest position in a memory window when the host component completes a transfer. This will increase the traffic to memory windows, but allow for a much smaller granularity in position updates.

In hindsight, this solution should have been the default behavior but to avoid backwards compatibility issues is added as a new capability controlled by an ABI  check. Only with a recent-enough Linux kernel will this behavior be enabled.

Reply:
@plbossart  Thanks a lot for providing this commit message. I think it would be better to understand.